### ISSUE TITLE
In IE 9 with JAWS 15, the contents of the field are not read out when tabbing back into it

### ISSUE THREAD

**Issue by paulwaitehomeoffice** [alphagov/accessible-autocomplete#365]
### Steps to reproduce

1. In IE, using JAWS, type some text into the field, and select an option
2. Tab out of the field
3. Tab back into the field

### Expected behaviour

JAWS reads out the contents of the field

### Actual behaviour

JAWS says “contains text”

### Notes

Tested in IE 9 with JAWS 15.

(In Edge (42) with JAWS 18, JAWS says “selected”, followed by the selected country, when tabbing back into the box. However, when typing in the box, it does not announce when suggestions appear, or that the user has to type two characters to get suggestions.)
